src/components/Table/Table.test.jsx

```diff
@@ -558,4 +560,179 @@ describe('Table', () => {
+    expect(screen.getAllByLabelText(i18nTest.columnSelectionButtonAria)[0]).toBeInTheDocument();
+    expect(screen.getAllByLabelText(i18nTest.filterButtonAria)[0]).toBeInTheDocument();
+    expect(screen.getAllByLabelText(i18nTest.editButtonAria)[0]).toBeInTheDocument();
+    expect(screen.getAllByText(i18nTest.searchLabel)[0]).toBeInTheDocument();
+    expect(screen.getAllByPlaceholderText(i18nTest.searchPlaceholder)[0]).toBeInTheDocument();
+    expect(screen.getAllByTitle(i18nTest.clearFilterAria)[0]).toBeInTheDocument();
+    expect(screen.getAllByLabelText(i18nTest.filterAria)[0]).toBeInTheDocument();
+    expect(screen.getAllByLabelText(i18nTest.openMenuAria)[0]).toBeInTheDocument();
+    expect(screen.getAllByText(i18nTest.batchCancel)[0]).toBeInTheDocument();
+    expect(screen.getByText(new RegExp(`.*\\s${i18nTest.itemSelected}.*`))).toBeInTheDocument();
+
+    expect(screen.queryByLabelText(i18nDefault.overflowMenuAria)).not.toBeInTheDocument();
+    expect(screen.queryByLabelText(i18nDefault.clickToExpandAria)).not.toBeInTheDocument();
+    expect(screen.queryByLabelText(i18nDefault.clickToCollapseAria)).not.toBeInTheDocument();
+    expect(screen.queryByLabelText(i18nDefault.selectAllAria)).not.toBeInTheDocument();
+    expect(screen.queryByLabelText(i18nDefault.selectRowAria)).not.toBeInTheDocument();
+    expect(screen.queryByLabelText(i18nDefault.downloadIconDescription)).not.toBeInTheDocument();
+
+    expect(screen.queryByText(i18nDefault.clearAllFilters)).not.toBeInTheDocument();
+    expect(screen.queryByLabelText(i18nDefault.columnSelectionButtonAria)).not.toBeInTheDocument();
+    expect(screen.queryByLabelText(i18nDefault.filterButtonAria)).not.toBeInTheDocument();
+    expect(screen.queryByLabelText(i18nDefault.editButtonAria)).not.toBeInTheDocument();
+    expect(screen.queryByText(i18nDefault.searchLabel)).not.toBeInTheDocument();
+    expect(screen.queryByPlaceholderText(i18nDefault.searchPlaceholder)).not.toBeInTheDocument();
+    expect(screen.queryByTitle(i18nDefault.clearFilterAria)).not.toBeInTheDocument();
+    expect(screen.queryByLabelText(i18nDefault.filterAria)).not.toBeInTheDocument();
+    expect(screen.queryByLabelText(i18nDefault.openMenuAria)).not.toBeInTheDocument();
+    expect(screen.queryByText(i18nDefault.batchCancel)).not.toBeInTheDocument();
+
+    rerender(
+      <Table
+        {...initialState}
+        options={{
+          ...initialState.options,
+          hasColumnSelectionConfig: true,
+        }}
+        i18n={i18nTest}
+        view={{
+          ...initialState.view,
+          toolbar: {
+            activeBar: 'column',
+          },
+        }}
+      />
+    );
+    expect(screen.getAllByText(i18nTest.columnSelectionConfig)[0]).toBeInTheDocument();
+    expect(screen.queryByText(i18nDefault.columnSelectionConfig)).not.toBeInTheDocument();
+  });
+
+  it('i18n string test 2', () => {
+    const i18nTest = {
```

Comment:
Not sure if these i18nTest objects are different but if not we can define them in the top of the file and reuse them in each test.